salesforce/management/commands/update_faculty_status.py

```diff
@@ -3,23 +3,31 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.models import Group
 from django.contrib.auth.models import User, Group
-
+from social.apps.django_app.default.models import DjangoStorage as SocialAuthStorage
 
 class Command(BaseCommand):
     help = "Add user to faculty group if confirmed by salesforce"
 
     def add_arguments(self, parser):
-        parser.add_argument('cms_id', type=int)
-        parser.add_argument('accounts_id', type=int)
-
+        parser.add_argument('cms_id', nargs='?', type=int,default=None)
+        parser.add_argument('--all', action='store_true', default=False)
     def handle(self, *args, **options):
+        if options['all']:
+            accounts_id = None
+        else:
+            social_user = SocialAuthStorage.user.objects.filter(user_id = options['cms_id'])
```

Comment:
I was just trying to explicitly state that the user I'd that is being filtered for is explicitly the primary key for the user in our django cms database and not accounts.  I can update it though.